packages/themes/src/hooks/useBreakpoint.hook.ts

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react';
```

Comment:
late in the day but is it possible to remove this hook entirely? 

I thought we agreed something similar in the UK - it doesn't run server side so I'm guessing (haven't tested) that initially user will get mobile view before it adjusts as the css will render before the js runs.

Would it be possible to just have 2 dedicated views, one for mobile/tablet and one for desktop and we just show and hide each one based only on css? (this would result in a bigger DOM but less reliance on js)

Reply:
@Nazzanuk If that's the move, then that's the move 🫡